tests/concurrency.py

```diff
@@ -10,6 +10,6 @@
 
 async def sleep(seconds: float) -> None:
     if sniffio.current_async_library() == "trio":
-        await trio.sleep(seconds)
+        await trio.sleep(seconds)  # pragma: nocover
```

Comment:
This `sleep()` method is only used from `conftest.TestServer`, but the server always runs under the `asyncio` loop so the `trio` branch is never executed.